core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -411,3 +411,8 @@ CFG_STM32_DEBUG_ACCESS ?= $(CFG_TEE_CORE_DEBUG)
 ifeq ($(call cfg-all-enabled,CFG_STM32MP15 CFG_STM32MP13),y)
 $(error CFG_STM32MP13_CLK and CFG_STM32MP15_CLK are exclusive)
 endif
+
+# Disable I2C trampoline if native I2C is used
+ifeq ($(CFG_STM32_I2C),y)
+$(call force,CFG_CORE_SE05X_I2C_TRAMPOLINE,n)
```

Comment:
Not strictly need. The switch is not default enbled in `mk/config.mk`, right?

Reply:
It is default to `y` in `core/drivers/crypto/se050/crypto.mk`, which gets included via `core/core.mk`.

Reply:
In _core/drivers/crypto/se050/crypto.mk_, `CFG_CORE_SE05X_I2C_TRAMPOLINE` defaults to `y` only when `CFG_NXP_SE05X` is enabled. So platforms not enabling `CFG_NXP_SE05X` should not bother with SE050 config directives.